css/style.css

```diff
@@ -0,0 +1,39 @@
+body{
+    padding: 0;
+    margin: 0;
+    background-color: #eff7f6;
+    text-align: center;
+    width: 100%;
+}
+header{
+    padding: 15px;
+}
+.logo{
+    margin-right: auto;
+    font-size: 25px;
+    font-weight: bold;
+}
+#nav-bar {
+    display: flex;
+    
+    border-bottom: 2px solid black;
+}
+ul{
+    display: flex;
+    list-style: none;
+    gap: 10px;
+}
+a {
+    text-decoration: none;
+    color: black;
```

Comment:
please it's better to use CSS classes than HTML tags.

Reply:
- [ ] Please the student needs to understand "WHY" are you requesting for change. Therefore you have to add an explanation. 
- [ ] Example: Please the container of this section must be a `<section>` tag instead of `<div>` (why?)because using a `<div>` does not define a section in HTML